web/src/components/historySection.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin } from '@brainhubeu/react-carousel';
+
+import HistoryItem from './historyItem';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+// const isBrowser = typeof window !== 'undefined';
+
+export default function History({ years }) {
+    const [activeTab, setActiveTab] = useState(years[0].id);
+    const [buttonComponentLogic, setButtonComponentLogic] = useState(false);
+
+    const setHistoryTimeline = () => {
```

Comment:
Do we need this? Coud we use `onLoad` to set the initial state?

Reply:
Actually, I think you will need this. Let's rename to something like `resizeHandler`